packages/manager/tools/registry/README.md

```diff
@@ -0,0 +1,210 @@
+manager-registry static --help
+Usage: manager-registry-static [options] [command]
+
+Options:
+  -V, --version                      output the version number
+  -h, --help                         output usage information
+
+Commands:
+  generate-manifests <registryPath>  Generate manifest for static registry
+  serve <registryPath>               Serve a static registry
+  add <registryPath> <fragmentPath>  Add a fragment in static registry
+  help [cmd]                         display help for [cmd]
+
+```
+
+##### Generate Manifests
+
+> Will generate and write manifest for registry and fragments.
+
+```sh
+$ manager-registry static generate-manifests --help
+Usage: manager-registry-static-generate-manifests [options] <registryPath>
+
+Options:
+  -V, --version                          output the version number
+  --fallbackRegistry <fallbackRegistry>  Fallback server registry url
+  -h, --help                             output usage information
+```
+
+*Options*
+
+* `--fallbackRegistry <fallbackRegistry>`: Fallback server registry url
+
+*Examples*
+
+```sh
+$ manager-registry static generate-manifests ./path/to/static/registry
+Manifests are generated for static registry in ./path/to/static/registry
+```
+
+To generate manifests with static fragments and fallback registry informations:
+
+```sh
+$ manager-registry static generate-manifests ./path/to/static/registry --fallbackRegistry http://localhost:1234
+Manifests are generated for static registry in ./path/to/static/registry with fallback informations from http://localhost:1234
+```
+
+
+##### Serve
+
+> Will serve a static registry
+
+```sh
+$ manager-registry static serve --help
+Usage: manager-registry-static-serve [options] <registryPath>
+
+Options:
+  -V, --version      output the version number
+  -p, --port <port>  server port (default: 8888)
+  -h, --help         output usage information
+```
+
+*Examples*
+
+```sh
+$ manager-registry static serve ./path/to/static/registry -p 1234
+Serve: ./path/to/static/registry - localhost:1234
+```
+
+##### Add fragment
+
+> Add a fragment dist to a static registry
+
+```sh
+$ manager-registry static add --help
+Usage: manager-registry-static-add [options] <registryPath> <fragmentPath>
+
+Options:
+  -V, --version  output the version number
+  -h, --help     output usage information
+```
+
+*Examples*
+
+Add a fragment dist (containing a `manifest.json`) in a static registry:
+
+```sh
+$ manager-registry static add ./path/to/static/registry ./path/to/fragment/dist
+Fragment "fragment@1.0.O" added in registry ./path/to/static/registry
+```
+
+Trying to add an existant fragment version in a static registry:
+
+```sh
+$ manager-registry static add ./path/to/static/registry ./path/to/fragment/dist
+Fragment "fragment@1.0.O" already exists in ./path/to/static/registry
+```
+
+
+## Related
+
+* [manager-registry](https://github.com/ovh-ux/manager/tree/master/packages/manager/tools/registry) - OVHcloud manager shared dev server configuration
```

Comment:
Wrong link I guess.

Isn't supposed to be:
```suggestion
* [dev-server-config](https://github.com/ovh/manager/tree/master/packages/manager/tools/dev-server-config) - OVHcloud manager shared dev server configuration
```

Reply:
Fixed.